db/migrate/20151435234636_add_container_node_max_container_group.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerNodeMaxContainerGroup < ActiveRecord::Migration
+  def change
+    add_column :container_nodes, :max_container_groups, :string
```

Comment:
Everything is exposed as string, it is json. We should handle types in the
way it makes sense. (Check for example ports) And i would assume internally
in k8s max is a number too.
On Sep 1, 2015 7:40 PM, "Ofer Schreiber" <notifications@github.com> wrote:

> In db/migrate/20151435234636_add_container_node_max_container_group.rb
> <https://github.com/ManageIQ/manageiq/pull/4138#discussion_r38441049>:
>
> > @@ -0,0 +1,5 @@
> > +class AddContainerNodeMaxContainerGroup < ActiveRecord::Migration
> > +  def change
> > +    add_column :container_nodes, :max_container_groups, :string
>
> @abonas <https://github.com/abonas> This is how k8s is exposing it in the
> api, I'm not sure we want to convert on our own
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/4138/files#r38441049>.
>